ui/client/components/toolbars/group/buttons/GroupButton.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react"
+import {useTranslation} from "react-i18next"
+import {useDispatch, useSelector} from "react-redux"
+import {groupSelected} from "../../../../actions/nk"
+import {ReactComponent as Icon} from "../../../../assets/img/toolbarButtons/group-finish.svg"
+import {canGroupSelection} from "../../../../reducers/graph/utils"
+import {getGraph} from "../../../../reducers/selectors/graph"
+import ToolbarButton from "../../../toolbarComponents/ToolbarButton"
+
+export function GroupButton(): JSX.Element {
+  const graph = useSelector(getGraph)
+  const {t} = useTranslation()
+  const dispatch = useDispatch()
+
+  return (
+    <ToolbarButton
```

Comment:
Why it's no longer Capabilities based?

Reply:
I agree with @mproch - why you removed it?

Reply:
no reason. mistake with merge.